.gitignore

```diff
@@ -4,3 +4,5 @@
 _output
 telepresence.log
 .openshift-tests-extension/openshift_payload_*.json
+report.json
+junit.xml
```

Comment:
Did you remove the junit part code? If so, we don't need them, and also need to update the PR title and description.

Reply:
not remove just for these files can be ignore during commit.